guess_the_outcome_rnn/Readme.md

```diff
@@ -0,0 +1,37 @@
+# RNN: Guess the outcome of a blackbox
+
+## Problem
+Consider a consistent program as a black-box, assume: 
+- For same inputs the result will always be the same (there is no impact of state or time)
+- The program may throw exceptions which can be considered as an output.
+- Many inputs may lead to same output.
+- Type(s) of input(s) are known.
+
+Using this system as a dataset, try to train a Recurrent Neural Network which is able to 
+guess the output.
+
+
+## Challenges 
+1. Training data is not present at the start of the exercise, this is an online training problem.
+2. Learning the challenges with training RNNs, discover the meaning of unstable in terms of training.
+3. Parameter tuning, controlling growth of weights as the training progresses.
+4. Creating a healthy batch of examples in a system where training data is available as a stream.
+
+
+## Tools
+- Pytorch 1.0.1
+- Python 3.5.6
```

Comment:
we actually might skip tools since someone might want to try something else. tools can go in a solution's readme not the kata's